cli/metrics/conn_other.go

```diff
@@ -1,7 +1,8 @@
-// +build !windows
+//go:build !windows,!darwin
+// +build !windows,!darwin
 
 /*
-   Copyright 2020 Docker Compose CLI authors
+   Copyright 2020, 2022 Docker Compose CLI authors
```

Comment:
I think the validation currently doesn't allow for the years to differ between files, so all should have the same years

Reply:
Yeah :( I even did a quick google for some more decent copyright header checker, but couldn't find one which does the right thing

Reply:
In https://github.com/docker/cli-docs-tool/, we've been trying Google addlicense, which seemed to work (a bit more "loosely" in checking); see https://github.com/docker/cli-docs-tool/pull/4